content/tokio/topics/tracing-next-steps.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Next steps with Tracing"
+---
+
+# Tokio-console
+
+Tokio-console is an htop-like utility that enables you to see a real-time view
+of an application’s spans and events. It can also represent "resources" that the
+Tokio runtime has created, such as Tasks. It's essential for understanding
+performance issues during the development process.
+
+To enable the console in mini-redis, you need to enable the `tracing` feature
+for the Tokio package:
```

Comment:
I am familiar enough with tokio/tracing but I never heard about mini-redis before looking at this PR... IMO we should add a link to the mini-redis repository, the very first time we mention it

```suggestion
For instance, to use Tokio-console in [the mini-redis project](https://github.com/tokio-rs/mini-redis)), 
you need to enable the `tracing` feature for the Tokio package:
```

Reply:
I agree we should add a link :+1:. Have you read the full guide? Most of the tutorial is based on mini-redis and includes a lot of links.

Reply:
> Have you read the full guide?

No, I did not take the time to do so 😅

I'm the kind of rustaceans who read the Rust book, then only read the relevant parts of the documentation of the crates I use... (if this is too difficult to use/understand a crate without reading its whole documentation, I'll look for another crate because the other members of my team won't take the time to read the whole documentation either...)

I guess Tokio is easy enough to use without having to read the "full guide", thank you for that :)

Reply:
thanks for the suggestion!